src/internal.c

```diff
@@ -3711,6 +3714,10 @@ static int DoKexDhReply(WOLFSSH* ssh, byte* buf, word32 len, word32* idx)
         ecc_key key_s;
     #endif
 #endif
+#ifndef WOLFSSH_NO_ECDH_NISTP256_KYBER_LEVEL1_SHA256
+    byte ss_hashSz = 0;
```

Comment:
Lets make variables follow camel case, right now we have a mix from not being strict enough. But our preference is camel case and the other snake case mix variable names will eventually be refactored.